skops/io/tests/test_external.py

```diff
@@ -427,3 +428,40 @@ def test_quantile_forest(self, quantile_forest, regr_data, trusted, tree_method)
         assert_method_outputs_equal(estimator, loaded, X)
 
         visualize(dumped, trusted=trusted)
+
+
+class TestSciKeras:
+    """Tests for SciKerasRegressor and SciKerasClassifier"""
+
+    @pytest.fixture(autouse=True)
+    def capture_stdout(self):
+        # Mock print and rich.print so that running these tests with pytest -s
+        # does not spam stdout. Other, more common methods of suppressing
+        # printing to stdout don't seem to work, perhaps because of pytest.
+        with patch("builtins.print", Mock()), patch("rich.print", Mock()):
+            yield
+
+    @pytest.fixture(autouse=True)
+    def keras(self):
+        scikeras = pytest.importorskip("scikeras")
+        return scikeras
+
+    @pytest.fixture
+    def test_dumping_model(self, keras):
+        from scikeras.wrappers import KerasClassifier
+
+        # This simplifies the basic usage tutorial from https://adriangb.com/scikeras/stable/notebooks/Basic_Usage.html
+
+        def get_clf(meta):
+            n_features_in_ = meta["n_features_in_"]
+            model = keras.models.Sequential()
+            model.add(keras.layers.Input(shape=(n_features_in_,)))
+            model.add(keras.layers.Dense(1, activation="sigmoid"))
+            return model
+
+        clf = KerasClassifier(model=get_clf, loss="binary_crossentropy")
+
+        pipeline = Pipeline([("classifier", clf)])
+
+        dump(clf, "keras-test.skops")
```

Comment:
Instead of just dumping, could we please do a cycle of `dumps` and `loads`, similar to the other tests?

Reply:
Done! I now have it dumping the model, loading it back in and comparing the results.

Reply:
Can we get rid of `dump` completely in favor of `dumps`? That way, we also don't need to care about cleaning up any files created during the test.

Reply:
Fixed! Sorry about that. I didn't realize the difference between `dumps` and `dump` 🤯